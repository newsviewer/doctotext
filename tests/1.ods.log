Using ODF/OOXML parser.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_TEXT command.
ODFOOXML_PARA command.
ODFOOXML_PARA command.
ODFOOXML_PARA command.
ODFOOXML_PARA command.
ODFOOXML_PARA command.
ODFOOXML_TABLE command.
ODFOOXML_PARA command.
ODFOOXML_URL command.
ODFOOXML_TABLE command.
ODFOOXML_TABLE command.
