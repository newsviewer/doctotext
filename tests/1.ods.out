_hyperlink_test_	







parsed links:
http://www.silvercoders.com/ @ index = 0 length = 16
